packages/app/src/components/search/SearchPage.tsx

```diff
@@ -129,6 +134,7 @@ const SearchPage = () => {
             </Grid>
           )}
           <Grid item xs>
+            <SearchResultLimiter className={classes.limiter} />
```

Comment:
Should this be added to create-app as well?

Reply:
Good question, what do you think @emmaindal ?

Reply:
yesss totally, thanks for adding it camila!